conda/recipes/cugraph-dgl/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - python x.x
   run:
     - cugraph ={{ version }}
-    - dgl >=0.9.1
+    - dgl >=1.1.0.dev0
```

Comment:
This is because post-1.0 DGL conda packages now have a cuda version suffix in the version, ex. `1.1.0.cu118`, see: https://anaconda.org/dglteam/dgl/labels. As a result, conda treats them as pre-release, so `dgl >=1.1.0` won't work in this case.

I wonder which of following is a better option:
- dgl >= 1.1.0.cu*
- dgl >=1.1.0.dev0

Reply:
Thanks, review addressed